src/Symfony/Component/Workflow/EventListener/GuardListener.php

```diff
@@ -55,6 +56,10 @@ private function getVariables(GuardEvent $event)
     {
         $token = $this->tokenStorage->getToken();
 
+        if (null === $token) {
+            throw new InvalidTokenConfigurationException('No token is set');
```

Comment:
Could maybe be something like `sprintf('There are no token available for the transition event %s', $eventName)` ?

WDYT @lyrixx ?

Reply:
So: 

`throw new InvalidTokenConfigurationException(sprintf('There are no token available for the transition event %s', $event->getWorkflowName()));`